tests/app/main/views/test_index.py

```diff
@@ -109,7 +96,7 @@ def test_when_document_is_unavailable(
 ):
     mocker.patch('app.service_api_client.get_service', return_value={'data': sample_service})
     mocker.patch('app.main.views.index._get_document_metadata', return_value=None)
-    response = client.get(
```

Comment:
Did you consider using `client.open(url, method=method)` here instead? I _think_ that should work and it feels a bit nicer. I always think using `getattr` should be a bit of a last resort (it's totally valid, but feels more obscure).

Changing this is totally optional :)

Reply:
I didn't know about `client.open()` but agree that is nicer - I've updated the code to use it and rebased the change into the relevant commit.